### PR TITLE
Update node version to 15.3.0 globally

### PR DESCRIPTION
**Pull request by aeifn**
These seem to be all places where the node version is specified.

### CODE REVIEW

File: .node-version [hunk 1]
```diff
@@ -1 +1 @@
-12.14.0
\ No newline at end of file
+15.3.0
```
Comment (kabalin) [inline .node-version:1]: Shall we rename this file .nvmrc? Looks like it was added specifically for use with `nvs`, but `nvs` supports `.nvmrc` also.

Comment (klimashkin) [reply .node-version:1]: If `.nvmrc` has wider adoption, then that is reasonable

Comment (kabalin) [reply .node-version:1]: Apparently looks like `.node-version` is [more generic](https://github.com/shadowspawn/node-version-usage), so let's keep as is, sorry for the noice.

Comment (aeifn) [reply .node-version:1]: Надо в README написать, как по-быстрому установить node manager или ссылку на quick start дать

**Review by klimashkin — APPROVED** [review]

**Pull request merged**